app/configurations/models.py

```diff
@@ -198,7 +198,7 @@ class CourseInformationMapping(TimeStampedModel):
                                                   " elasticsearch")
 
     course_derived_from = models.CharField(max_length=200,
-                                           default="P2881_Core.DerivedFrom",
+                                           default="P2881-Core.DerivedFrom",
```

Comment:
@mdonlon don't bother changing it back, but you don't typically need to change the default, as it is just the pre-populated value for the field when you go to create an instance of it

Reply:
Noted